src/utilities/flattenTopLevelFields.ts

```diff
@@ -10,17 +10,19 @@ import {
 
 const flattenFields = (fields: Field[], keepPresentationalFields?: boolean): (FieldAffectingData | FieldPresentationalOnly)[] => {
   return fields.reduce((fieldsToUse, field) => {
-    if (fieldAffectsData(field) || (keepPresentationalFields && fieldIsPresentationalOnly(field))) {
-      return [
-        ...fieldsToUse,
-        field,
-      ];
-    }
-
     if (fieldHasSubFields(field)) {
```

Comment:
Can this function return the subfields so you don't have to iterate fields again on line 14? I haven't looked at this function but I imagine it does a very similar thing.

Reply:
Jim has said to hold off for now, we will do this as a feature request.